aloha-vw-jni/src/main/scala/com/eharmony/aloha/models/vw/jni/multilabel/VwMultilabelParamAugmentation.scala

```diff
@@ -0,0 +1,453 @@
+package com.eharmony.aloha.models.vw.jni.multilabel
+
+import java.io.File
+
+import com.eharmony.aloha.dataset.vw.multilabel.VwMultilabelRowCreator
+import com.eharmony.aloha.dataset.vw.multilabel.VwMultilabelRowCreator.LabelNamespaces
+import com.eharmony.aloha.models.vw.jni.multilabel.VwSparseMultilabelPredictor.ExpectedLearner
+import org.apache.commons.io.{FileUtils, IOUtils}
+import vowpalWabbit.learner.VWLearners
+
+import scala.util.matching.Regex
+import scala.util.{Failure, Success, Try}
+
+
+/**
+  * Created by ryan.deak on 10/5/17.
+  */
+protected trait VwMultilabelParamAugmentation {
+
+  protected type VWNsSet = Set[Char]
+  protected type VWNsCrossProdSet = Set[(Char, Char)]
+
+  /**
+    * Adds VW parameters to make the parameters work as an Aloha multilabel model.
```

Comment:
I would add a comment here about how this is experimental and tied to a specific version of VW.